src/commonMain/kotlin/fr/acinq/lightning/channel/states/Normal.kt

```diff
@@ -192,11 +194,10 @@ data class Normal(
                                 is InteractiveTxSigningSessionAction.SendTxSigs -> sendSpliceTxSigs(spliceStatus.origins, action, cmd.message.channelData)
                             }
                         }
-                        commitments.params.channelFeatures.hasFeature(Feature.DualFunding) && commitments.latest.localFundingStatus.signedTx == null && cmd.message.batchSize == 1 -> {
-                            // The latest funding transaction is unconfirmed and we're missing our peer's tx_signatures: any commit_sig that we receive before that should be ignored,
-                            // it's either a retransmission of a commit_sig we've already received or a bug that will eventually lead to a force-close anyway.
-                            // NB: we check the dual-funding feature, because legacy single-funding unconfirmed channels will have `localFundingStatus=UnconfirmedFundingTx`,
-                            // so we cannot just check on the absence of `signedTx`.
+                        commitments.params.channelFeatures.hasFeature(Feature.DualFunding) && ignoreRetransmittedCommitSig(cmd.message) -> {
```

Comment:
If you call the method `ignoreRetransmittedCommitSig`, then it means that we are in a dual-funding context and we should move the feature check to that method imo, makes things lighter on the calling site:

```suggestion
                        ignoreRetransmittedCommitSig(cmd.message) -> {
```

Reply:
Agreed, done in https://github.com/ACINQ/lightning-kmp/pull/536/commits/d515fc598801e065728d4a2212db885ee2e23336